src/pattern_library/css/blocks/pagination.css

```diff
@@ -0,0 +1,31 @@
+.pagination {
```

Comment:
I remember your comment from the figjam file and I agree that we might/should use "hollow button" design here instead for greater consistency.

This leads me to questions for the knowledge sharing session:
- When/how do we decide to create variants (with data attributes) of a block?
- What is the "inheritance hierarchy" of the blocks, i.e. when do you create a new block, copying styles from another one? Maybe "inheriting" is strictly forbidden. 
- How do we avoid duplication in this system?
- Is there something like a "code smell" in this library, some rough guideline of "if we catch ourselves doing X then that's an indication of Y and we should be doing Z instead"?